scripts/EN/Harry

```diff
@@ -9107,47 +9107,47 @@ L_0xb9b2:
+ShowText((1315)"¥"Damn…!¥"¥nI kick it in the face with the bottom of my shoe, W1and it falls down on its back. W1It then begins to stand for its next attack."); // 0xba40: 0000100c
 HandleInput(); // 0xba46: 00000010
 CleanScreen(); // 0xba4a: 00000064
 DrawBackground(); // 0xba4e: 00000065
 Jump(L_0xc0f4); // 0xba52: 00001001
 L_0xba58:
 OP_0xe(0x7a); // 0xba58: 0000100e
-ShowText((1439)"I made my way into the room at the end of the passage. It is small, and the smell of something having been burned is in the air. Once I saw an altar appear in the darkness I realized that this is the ¥"church¥" that Dahlia was talking about."); // 0xba5e: 0000100c
+ShowText((1439)"I made my way into the room at the end of the passage.¥nW2It is small, W1and the smell of something having been burned is in the air.¥nW2Once I saw an altar appear in the darkness W1I realized that this is the W1¥"church¥"W1 that Dahlia was talking about."); // 0xba5e: 0000100c
 HandleInput(); // 0xba64: 00000010
-ShowText((1440)"I look around the area. An axe have been displayed on the wall. But nothing else stands out. While shaking my head I decide to try going back through the passage I came from. And at that moment...a light bright enough to make me look away emanates from the altar."); // 0xba68: 0000100c
+ShowText((1440)"I look around the area.¥nW2An axe have been displayed on the wall.¥nW2But nothing else stands out. ¥nW2While shaking my head I decide to try going back through the passage I came from.¥n¥nW2And at that moment…W5a light bright enough to make me look away emanates from the altar."); // 0xba68: 0000100c
 HandleInput(); // 0xba6e: 00000010
 OP_0x1b(0x14); // 0xba72: 0000101b
-ShowText((1441)"I think I heard the sound of a giant match being stuck, and then two enormous pillars of flame rose from the altar, as if a ritual were beginning or something. Even at the distance I am at I can feel the heat from those flames."); // 0xba78: 0000100c
+ShowText((1441)"I think I heard the sound of a giant match being stuck, W1and then two enormous pillars of flame rose from the altar, W1as if a ritual were beginning or something. W1Even at the distance I am at I can feel the heat from those flames."); // 0xba78: 0000100c
 HandleInput(); // 0xba7e: 00000010
 OP_0x1b(0x15); // 0xba82: 0000101b
 Jump(L_0xc0f4); // 0xba88: 00001001
 L_0xba8e:
 LoadSecondEffect(0x23, 0x1, 0x1); // 0xba8e: 0000306e
-ShowText((1475)"It is a giant caterpillar about 5 meters long. It sprays a green liquid from its fierce tusk-covered mouth. I can tell that the liquid is poisonous just from looking at it. And that it is definitely being discharged towards me."); // 0xba98: 0000100c
+ShowText((1475)"It is a giant caterpillar about 15 feet long.¥nW2It sprays a green liquid from its fierce tusk-covered mouth. W1I can tell that the liquid is poisonous just from looking at it. ¥nW2And that it is definitely being discharged towards me."); // 0xba98: 0000100c
```

Comment:
Changed 5 meters to 15 feet. 
It is closer to 16.4, but 15 has a good sound to it : )